README.md

```diff
@@ -26,7 +26,7 @@ JSON format is described using [MSON](https://github.com/apiaryio/mson).
     - [Document structure](#document-structure)
         - [HAL document form extension (object)](#hal-document-form-extension-object)
             - [Properties](#properties)
-        - [Form (object)](#form-object)
+        - [Form (object)](#form-
```

Comment:
Looks like the trailing `)` is missing here.

Reply:
good catch. thanks.